textproc/lttoolbox/Portfile

```diff
@@ -3,6 +3,10 @@
 PortSystem              1.0
 PortGroup               cmake 1.1
 PortGroup               github 1.0
+PortGroup               legacysupport 1.1
+
+# asprintf_l, macos 10.7
+legacysupport.newest_darwin_requires_legacy 11
```

Comment:
That won't help.

You need to append the compilation flag `-D__DARWIN_C_LEVEL=__DARWIN_C_FULL` for macOS 10.7 and 10.8, similar to [what I did](https://github.com/macports/macports-ports/commit/d85d6c821adba90638ec280b5445e2fba1f17e45) with `knot-resolver`.

I also see in the logs that `-std=c++2b` flag is being used, so it might be worth bump `compiler.cxx_standard` to `2020`.